CHANGES

```diff
@@ -9,6 +9,9 @@
 
  Changes between 1.1.0f and 1.1.1 [xx XXX xxxx]
 
+  *) Add TLSv1.3 post-handshake authentication.
```

Comment:
As an aside, we haven't actually got a CHANGES entry for adding TLSv1.3 support yet!! I should probably create a PR.

Reply:
I can remove this, if you'd prefer a "bigger" TLSv1.3 addition to `CHANGES`

Reply:
No - this was an aside only. I will add a separate TLSv1.3 entry, but there's no reason not to have this one too.